bin/bam2cov_filt.py

```diff
@@ -72,7 +72,7 @@ def bam_parser(bam_file):
 
 def FP_control(out_root, genomes_len, unique_matches, ave_read_len):
     unique_thres01 = []
-    cov_threhold = 0.1
+    cov_threhold = 0.01
```

Comment:
```suggestion
    COV_THREHOLD = 0.01
```
Using upper case is a convention for constants

Reply:
Added.